amr-wind/utilities/index_operations.H

```diff
@@ -19,6 +19,17 @@ closest_index(const amrex::Vector<amrex::Real>& vec, const amrex::Real value)
     return std::max(idx - 1, 0);
 }
 
+//! This version is required for the ERF coupling
```

Comment:
remove comment, use comment from the function above, modify it slightly. Also, while you are at it, can you rename `closest_index` to `closest_index_ubound` ? That way everything is explicit.